snippets/standard/datetime/json/serializing-with-jsonserializer.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Text.Json;
+
+public class Example
+{
+    private class Product
+    {
+        public string Name { get; set; }
+        public DateTime ExpiryDate { get; set; }
+    }
+
+    public static void Main(string[] args)
+    {
+        Product p = new Product();
+        p.Name = "Banana";
+        p.ExpiryDate = new DateTime(2019, 7, 26);
+        string json = JsonSerializer.Serialize(p);
```

Comment:
nit: consider adding a line break between product p init and the call to serialize.